src/helpers/fetchWithTimeout.ts

```diff
@@ -0,0 +1,25 @@
+import fetch from 'cross-fetch'
+
+export const fetchWithTimeout = async (
+  url: string,
+  body: any | null = null,
+  duration: number,
```

Comment:
```suggestion
  duration: number,
  body?: RequestInit
```
using "?" instead of defaulting to null is a nit. But using RequestInit instead of any is not. We should try to keep the benefits of static typing wherever possible instead of using `any`